app/graphql/mutations/plans/create.rb

```diff
@@ -18,6 +18,7 @@ class Create < BaseMutation
       argument :name, String, required: true
       argument :parent_id, ID, required: false
       argument :pay_in_advance, Boolean, required: true
+      argument :tax_codes, [String], required: false
```

Comment:
Why not receive an array `Types::Taxes::Object`, as this is what the pan query returns?

Reply:
to be consistent with the API... is that a blocker for you?